src/main/java/bookrepo/model/User.java

```diff
@@ -32,4 +40,46 @@ public class User {
     private String shippingAddress;
     @Column(nullable = false)
     private boolean isDeleted;
+    @ManyToMany(fetch = FetchType.EAGER)
```

Comment:
avoid using `FetchType.EAGER`
```suggestion
    @ManyToMany
```